src/styles.scss

```diff
@@ -57,53 +130,51 @@ ul {
 }
 
 .main-container {
-  width: 100%;
-  // height: calc(100% - 60px);
+  // width: 100%;
+  height: calc(100% - 60px);
   height: 660px;
   padding: 1.5rem;
 }
```

Comment:
commented code, please remove it.
also  `height: 660px;` overwrites ` height: calc(100% - 60px);`
and it is really not a good practice to provide height and width  in pixels. better use %, vw, vh, etc (you can probably use pixels for font-size, button height, ...)